_templates/countdown.php

```diff
@@ -0,0 +1,9 @@
+<script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
```

Comment:
We load it asynchronously, so you need to wrap `scripts/countdown.js` with the usual jQuery onload parameter that only gets fired once it's there.